From: Graeme Foot <graeme.foot@touchcut.com>
Date: Mon, 01 Feb 2018 11:00:00 +1200

Give the wake variable a default value to avoid uninitialised variable warning

diff --git a/devices/e1000/e1000_main-2.6.31-ethercat.c b/devices/e1000/e1000_main-2.6.31-ethercat.c
--- a/devices/e1000/e1000_main-2.6.31-ethercat.c
+++ b/devices/e1000/e1000_main-2.6.31-ethercat.c
@@ -4921,7 +4921,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-2.6.32-ethercat.c b/devices/e1000/e1000_main-2.6.32-ethercat.c
--- a/devices/e1000/e1000_main-2.6.32-ethercat.c
+++ b/devices/e1000/e1000_main-2.6.32-ethercat.c
@@ -4762,7 +4762,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-2.6.33-ethercat.c b/devices/e1000/e1000_main-2.6.33-ethercat.c
--- a/devices/e1000/e1000_main-2.6.33-ethercat.c
+++ b/devices/e1000/e1000_main-2.6.33-ethercat.c
@@ -4787,7 +4787,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-2.6.35-ethercat.c b/devices/e1000/e1000_main-2.6.35-ethercat.c
--- a/devices/e1000/e1000_main-2.6.35-ethercat.c
+++ b/devices/e1000/e1000_main-2.6.35-ethercat.c
@@ -4820,7 +4820,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-2.6.37-ethercat.c b/devices/e1000/e1000_main-2.6.37-ethercat.c
--- a/devices/e1000/e1000_main-2.6.37-ethercat.c
+++ b/devices/e1000/e1000_main-2.6.37-ethercat.c
@@ -4916,7 +4916,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.0-ethercat.c b/devices/e1000/e1000_main-3.0-ethercat.c
--- a/devices/e1000/e1000_main-3.0-ethercat.c
+++ b/devices/e1000/e1000_main-3.0-ethercat.c
@@ -4962,7 +4962,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.10-ethercat.c b/devices/e1000/e1000_main-3.10-ethercat.c
--- a/devices/e1000/e1000_main-3.10-ethercat.c
+++ b/devices/e1000/e1000_main-3.10-ethercat.c
@@ -5274,7 +5274,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.12-ethercat.c b/devices/e1000/e1000_main-3.12-ethercat.c
--- a/devices/e1000/e1000_main-3.12-ethercat.c
+++ b/devices/e1000/e1000_main-3.12-ethercat.c
@@ -5274,7 +5274,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.14-ethercat.c b/devices/e1000/e1000_main-3.14-ethercat.c
--- a/devices/e1000/e1000_main-3.14-ethercat.c
+++ b/devices/e1000/e1000_main-3.14-ethercat.c
@@ -5256,7 +5256,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.16-ethercat.c b/devices/e1000/e1000_main-3.16-ethercat.c
--- a/devices/e1000/e1000_main-3.16-ethercat.c
+++ b/devices/e1000/e1000_main-3.16-ethercat.c
@@ -5248,7 +5248,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.18-ethercat.c b/devices/e1000/e1000_main-3.18-ethercat.c
--- a/devices/e1000/e1000_main-3.18-ethercat.c
+++ b/devices/e1000/e1000_main-3.18-ethercat.c
@@ -5372,7 +5372,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.4-ethercat.c b/devices/e1000/e1000_main-3.4-ethercat.c
--- a/devices/e1000/e1000_main-3.4-ethercat.c
+++ b/devices/e1000/e1000_main-3.4-ethercat.c
@@ -5260,7 +5260,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.6-ethercat.c b/devices/e1000/e1000_main-3.6-ethercat.c
--- a/devices/e1000/e1000_main-3.6-ethercat.c
+++ b/devices/e1000/e1000_main-3.6-ethercat.c
@@ -5274,7 +5274,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-3.8-ethercat.c b/devices/e1000/e1000_main-3.8-ethercat.c
--- a/devices/e1000/e1000_main-3.8-ethercat.c
+++ b/devices/e1000/e1000_main-3.8-ethercat.c
@@ -5289,7 +5289,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-4.1-ethercat.c b/devices/e1000/e1000_main-4.1-ethercat.c
--- a/devices/e1000/e1000_main-4.1-ethercat.c
+++ b/devices/e1000/e1000_main-4.1-ethercat.c
@@ -5371,7 +5371,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-4.13-ethercat.c b/devices/e1000/e1000_main-4.13-ethercat.c
--- a/devices/e1000/e1000_main-4.13-ethercat.c
+++ b/devices/e1000/e1000_main-4.13-ethercat.c
@@ -5383,7 +5383,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-4.14-ethercat.c b/devices/e1000/e1000_main-4.14-ethercat.c
--- a/devices/e1000/e1000_main-4.14-ethercat.c
+++ b/devices/e1000/e1000_main-4.14-ethercat.c
@@ -5401,7 +5401,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-4.4-ethercat.c b/devices/e1000/e1000_main-4.4-ethercat.c
--- a/devices/e1000/e1000_main-4.4-ethercat.c
+++ b/devices/e1000/e1000_main-4.4-ethercat.c
@@ -5369,7 +5369,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
diff --git a/devices/e1000/e1000_main-4.9-ethercat.c b/devices/e1000/e1000_main-4.9-ethercat.c
--- a/devices/e1000/e1000_main-4.9-ethercat.c
+++ b/devices/e1000/e1000_main-4.9-ethercat.c
@@ -5400,7 +5400,7 @@ static int e1000_resume(struct pci_dev *
 
 static void e1000_shutdown(struct pci_dev *pdev)
 {
-	bool wake;
+	bool wake = false;
 
 	__e1000_shutdown(pdev, &wake);
 
